CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.10.0]
```

Comment:
I've been wondering about making a quick PR to make the changelog messages for `0.10.0` (hopefully) a little easier to read. Happy for me to do that?

Reply:
> I've been wondering about making a quick PR to make the changelog messages for 0.10.0 (hopefully) a little easier to read.

OK :+1:  but it's easier for me if you specify what should be changed here.

Reply:
Or maybe you can push the change directly to this branch.

Reply:
:+1: will push later